site/installing-weave.md

```diff
@@ -36,6 +36,12 @@ check, run the following before launching Weave Net:
 
     export CHECKPOINT_DISABLE=1
 
+The information sent in this check is:
```

Comment:
"host UUID hash"? "anonymized host UUID"

Basically I think it's important to communicate that we can tie the information to a single, albeit anonymised, host.